common/src/test/java/io/netty/util/internal/StringUtilTest.java

```diff
@@ -322,6 +322,61 @@ private static void escapeCsv(CharSequence value, CharSequence expected) {
     }
 
     @Test
+    public void unescapeCsv() {
+        assertEquals("", StringUtil.unescapeCsv(""));
```

Comment:
There is an `escapeCsv` method in `StringUtilTest`. So I just used the static import for unescapeCsv.